app/webpack/javascripts/modules/Modules.AllocationDataTable.js

```diff
@@ -59,6 +59,10 @@ moj.Modules.AllocationDataTable = {
         $(row).addClass('cav_warning');
         $('td', row).eq(0).wrapInner("<div class='warning-message-container'></div>");
         $('td .warning-message-container', row).eq(0).append("<div class='warning-message'>CAVs not injected</div>");
+      } else if (data.filter.scheme_12_warning) {
```

Comment:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3478/files#r476385484)

Reply:
👍 have amended naming to scheme_12_fees_warning